### PR TITLE
French localization now available.

### PR DESCRIPTION
**Pull request by dlepage**
One more language...

### CODE REVIEW

File: config/locales/fr.yml [hunk 1]
```diff
@@ -0,0 +1,197 @@
+fr:
```
Comment (parndt) [inline config/locales/fr.yml:1]: this should be `fr:`

Comment (parndt) [inline config/locales/fr.yml:1]: which I'm guessing you just noticed at the same time!

**Comment by dlepage** [comment]
fixed ! Sorry

**Comment by parndt** [comment]
Merged in cd49c29 thank you!

File: config/locales/fr.yml (hunk 1 above)
Comment (dlepage) [inline config/locales/fr.yml:1]: You bet ! :)

**Pull request closed without merging**